ATAOS/v3/hex_m1_hex_202102_v3.yaml

```diff
@@ -2,6 +2,7 @@
 # Date created: 2021-02-16
 # Description: DM-28826 Support AT observing run
```

Comment:
this needs more description. How does it differ from 20210218 ? When is one used versus the other?